2021-01-09 17:01:18,527 - root - INFO - main.py:53 - START SERVER
2021-01-09 17:01:18,668 - root - INFO - gateway_server.py:186 - 服务端(('192.168.8.102', 8000))已成功连接。。online
2021-01-09 17:01:21,703 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:01:21,703 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:01:21,833 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:01:21,935 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:01:22,038 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:01:22,149 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:01:22,252 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:01:22,363 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:01:23,045 - root - WARNING - gateway_server.py:105 - equipment (192.168.8.102, 8000) is offline
2021-01-09 17:01:23,064 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:01:42,484 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:01:45,543 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:01:45,543 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:01:45,704 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:01:45,714 - root - INFO - gateway_server.py:186 - 服务端(('192.168.8.102', 8000))已成功连接。。online
2021-01-09 17:01:45,845 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:01:45,851 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:45,852 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:01:45,852 - root - INFO - base_adapter.py:74 - SDK初始化成功
2021-01-09 17:01:45,857 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:45,857 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:01:45,859 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:45,859 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:01:45,955 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:45,956 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:01:45,958 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:45,958 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:01:45,959 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:45,961 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:01:45,999 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:45,999 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:01:46,140 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:01:56,069 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:56,075 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:01:56,087 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:56,088 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:01:56,095 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:56,095 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:01:56,096 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:56,096 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:01:56,101 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:56,108 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:01:56,110 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:56,110 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:01:56,111 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:56,111 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:01:56,123 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:01:56,124 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:02:06,000 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:02:06,157 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:02:06,165 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:06,165 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:02:06,167 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:06,168 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:06,169 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:06,169 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:06,170 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:06,170 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:06,172 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:06,172 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:06,174 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:06,180 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:06,183 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:06,183 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:02:06,184 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:06,186 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:02:09,187 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:02:09,187 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:02:09,353 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:02:09,558 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:02:16,241 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:16,241 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:02:16,242 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:16,253 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:02:16,254 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:16,255 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:16,256 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:16,256 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:16,263 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:16,280 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:16,288 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:16,289 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:16,291 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:16,291 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:16,292 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:16,292 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:02:16,294 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:16,294 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:02:26,344 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:26,345 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:02:26,345 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:26,346 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:26,347 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:26,347 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:26,347 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:26,348 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:26,349 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:26,349 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:26,350 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:26,351 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:26,351 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:26,351 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:02:26,354 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:26,354 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:02:29,505 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:02:29,623 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:02:32,648 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:02:32,648 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:02:32,853 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:02:33,001 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:02:36,406 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,406 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:02:36,407 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,407 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:36,420 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,421 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:36,421 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,421 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:36,422 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,422 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:36,423 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,423 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:02:36,424 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,424 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:02:36,425 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,425 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:02:36,426 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,426 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:02:36,426 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,426 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:02:36,427 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,427 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:02:36,428 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,428 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:36,436 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,436 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:02:36,476 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:36,477 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:02:46,508 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:46,508 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:02:46,510 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:46,510 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:46,517 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:46,517 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:46,524 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:46,536 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:46,552 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:46,552 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:46,559 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:46,559 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:02:46,569 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:46,570 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:02:53,018 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:02:53,159 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:02:56,204 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:02:56,205 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:02:56,338 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:02:56,480 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:02:56,863 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:56,863 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:02:56,864 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:56,864 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:56,871 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:56,871 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:56,878 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:56,878 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:56,879 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:56,879 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:02:56,892 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:56,892 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:02:56,894 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:02:56,895 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:03:06,940 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:06,940 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:03:06,943 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:06,944 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:06,947 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:06,949 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:06,953 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:06,963 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:06,965 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:06,966 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:06,972 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:06,984 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:03:07,000 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:07,000 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:03:16,512 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:03:16,663 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:03:17,047 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:17,059 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:03:17,066 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:17,071 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:17,084 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:17,097 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:17,115 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:17,115 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:17,116 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:17,116 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:17,117 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:17,117 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:03:17,119 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:17,119 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:03:19,706 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:03:19,713 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:03:19,900 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:03:19,937 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:03:27,164 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:27,165 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:03:27,166 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:27,166 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:27,167 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:27,167 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:27,168 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:27,168 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:27,179 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:27,179 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:27,191 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:27,191 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:03:27,193 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:27,193 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:03:37,232 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:37,232 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:03:37,233 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:37,233 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:37,235 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:37,235 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:37,239 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:37,240 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:37,243 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:37,243 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:37,244 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:37,244 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:03:37,246 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:37,246 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:03:40,071 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:03:40,209 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:03:43,259 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:03:43,263 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:03:43,316 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:03:43,427 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:03:47,297 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:47,297 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:03:47,298 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:47,298 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:47,299 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:47,299 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:47,300 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:47,300 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:47,301 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:47,301 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:47,310 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:47,310 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:03:47,346 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:47,346 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:03:57,403 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:57,405 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:03:57,406 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:57,406 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:57,407 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:57,407 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:57,408 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:57,408 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:57,412 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:57,412 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:03:57,418 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:57,418 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:03:57,426 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:03:57,426 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:03,574 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:04:03,681 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:04:06,695 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:04:06,708 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:04:06,782 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:04:06,848 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:04:07,503 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:07,504 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:07,517 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:07,518 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:07,519 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:07,519 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:07,520 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:07,520 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:07,521 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:07,521 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:07,521 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:07,522 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:04:07,523 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:07,523 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:17,557 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:17,568 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:04:17,575 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:17,581 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:17,625 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:17,633 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:17,634 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:17,634 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:17,635 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:17,635 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:17,640 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:17,640 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:04:17,642 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:17,642 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:26,995 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:04:27,144 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:04:27,716 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:27,716 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:27,717 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:27,717 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:27,718 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:27,718 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:27,719 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:27,719 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:27,720 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:27,720 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:27,730 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:27,737 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:27,744 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:27,744 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:04:27,752 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:27,752 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:27,789 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:27,789 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:04:27,803 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:27,810 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:28,805 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:28,806 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:04:28,808 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:28,808 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:28,833 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:28,833 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:28,857 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:28,862 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:04:28,869 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:28,869 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:28,905 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:28,905 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:04:28,906 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:28,906 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:04:28,914 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:28,916 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:30,186 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:04:30,187 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:04:30,333 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:04:30,484 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:04:39,209 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:39,209 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:39,210 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:39,210 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:39,217 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:39,223 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:39,225 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:39,225 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:39,231 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:39,237 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:39,242 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:39,242 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:39,244 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:39,244 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:04:39,246 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:39,246 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:49,333 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:49,334 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:04:49,336 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:49,336 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:04:49,337 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:49,337 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:04:49,337 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:49,338 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:49,353 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:49,359 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:49,365 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:49,385 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:49,399 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:49,435 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:49,448 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:49,455 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:49,468 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:49,475 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:04:49,477 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:49,478 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:50,494 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:04:50,652 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:04:53,692 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:04:53,698 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:04:53,849 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:04:53,949 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:04:59,527 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:59,527 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:04:59,528 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:59,528 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:59,530 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:59,530 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:59,543 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:59,545 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:59,553 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:59,554 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:59,555 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:59,555 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:04:59,563 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:59,570 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:04:59,594 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:04:59,594 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:05:09,655 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:09,661 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:05:09,662 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:09,663 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:09,664 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:09,664 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:09,666 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:09,666 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:09,667 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:09,667 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:09,668 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:09,668 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:09,670 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:09,670 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:05:09,671 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:09,671 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:05:14,007 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:05:14,132 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:05:17,146 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:05:17,146 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:05:17,274 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:05:17,449 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:05:19,698 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:19,699 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:05:19,700 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:19,700 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:19,706 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:19,712 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:19,742 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:19,743 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:19,744 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:19,744 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:19,746 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:19,746 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:19,769 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:19,770 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:05:19,772 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:19,772 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:05:29,812 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:29,831 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:05:29,837 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:29,837 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:29,845 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:29,845 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:29,846 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:29,846 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:29,855 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:29,855 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:29,862 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:29,862 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:29,869 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:29,869 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:05:29,871 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:29,872 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:05:37,412 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:05:37,538 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:05:39,899 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:39,899 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:05:39,900 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:39,900 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:39,916 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:39,922 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:39,930 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:39,943 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:39,945 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:39,945 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:39,952 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:39,952 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:05:39,986 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:40,003 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:05:40,567 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:05:40,585 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:05:40,800 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:05:40,835 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:05:50,158 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:50,164 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:05:50,165 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:50,165 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:50,178 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:50,178 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:50,183 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:50,184 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:50,185 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:50,185 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:05:50,186 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:50,186 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:05:50,192 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:05:50,192 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:00,262 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:00,262 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:00,275 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:00,275 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:00,298 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:00,312 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:00,327 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:00,330 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:00,331 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:00,332 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:00,333 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:00,333 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:06:00,335 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:00,335 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:00,941 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:06:01,058 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:06:04,105 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:06:04,111 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:06:04,232 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:06:04,420 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:06:10,389 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:10,389 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:06:10,390 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:10,391 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:10,392 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:10,398 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:10,423 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:10,423 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:10,439 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:10,440 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:10,466 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:10,466 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:10,480 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:10,481 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:06:10,502 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:10,502 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:10,554 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:10,554 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:10,585 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:10,585 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:11,566 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,566 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:06:11,568 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,568 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:11,591 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,591 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:06:11,593 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,598 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:11,600 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,600 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:06:11,631 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,632 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:11,702 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,702 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:11,703 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,703 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:06:11,705 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,705 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1001
2021-01-09 17:06:11,707 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,707 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:11,714 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,714 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:06:11,727 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:11,738 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:22,016 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:22,017 - root - INFO - base_adapter.py:59 - 执行成功,返回值：0
2021-01-09 17:06:22,018 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:22,018 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:22,019 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:22,019 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:22,045 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:22,045 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:22,053 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:22,054 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:22,056 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:22,056 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:22,057 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:22,057 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:06:22,064 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:22,073 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:24,366 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.118', 23)
2021-01-09 17:06:24,485 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 26)
2021-01-09 17:06:27,506 - root - WARNING - Object2.py:861 - Fail to init Entreance_zk connect (192.168.0.201, 4370)
2021-01-09 17:06:27,506 - root - INFO - gateway_server.py:186 - 服务端(('192.168.0.201', 4370))已成功连接。。online
2021-01-09 17:06:27,688 - root - WARNING - gateway_server.py:194 - fail to connect to server ('192.168.0.117', 23)
2021-01-09 17:06:27,931 - root - WARNING - gateway_server.py:105 - equipment (192.168.0.201, 4370) is offline
2021-01-09 17:06:32,126 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:32,126 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
2021-01-09 17:06:32,127 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:32,127 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:32,132 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:32,132 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:32,133 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:32,133 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:32,136 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:32,151 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:32,154 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:32,154 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1000
2021-01-09 17:06:32,161 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:32,168 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1002
2021-01-09 17:06:32,183 - root - INFO - base_adapter.py:58 - 调用的库：/home/alex/PycharmProjects/ToolManageSys/util/hkws/../libs/hkvision_lib/libhcnetsdk.so
2021-01-09 17:06:32,183 - root - INFO - base_adapter.py:59 - 执行成功,返回值：1
